<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1459</width>
    <height>1041</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>10</y>
      <width>1021</width>
      <height>1011</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLineEdit" name="Player_name"/>
         </item>
         <item>
          <widget class="QPushButton" name="SearchPlayer">
           <property name="text">
            <string>search</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="player_info">
         <property name="rowCount">
          <number>1</number>
         </property>
         <property name="columnCount">
          <number>9</number>
         </property>
         <row/>
         <column>
          <property name="text">
           <string>Player Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Team Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Points Scored</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Rebounds</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Assists</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Blocks</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Field Goal Percentage</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Seconds</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>                                            player stats</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="Reboundstab">
       <property name="currentIndex">
        <number>4</number>
       </property>
       <widget class="QWidget" name="players_2">
        <attribute name="title">
         <string>Points</string>
        </attribute>
        <widget class="QTableWidget" name="tableWidget_3">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>10</y>
           <width>321</width>
           <height>192</height>
          </rect>
         </property>
         <property name="rowCount">
          <number>20</number>
         </property>
         <property name="columnCount">
          <number>6</number>
         </property>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <column>
          <property name="text">
           <string>Rank</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Player</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Points</string>
          </property>
         </column>
         <column/>
         <column/>
         <column/>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>Rebounds</string>
        </attribute>
        <widget class="QTableWidget" name="tableWidget_4">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>10</y>
           <width>321</width>
           <height>192</height>
          </rect>
         </property>
         <property name="rowCount">
          <number>20</number>
         </property>
         <property name="columnCount">
          <number>6</number>
         </property>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <column>
          <property name="text">
           <string>Rank</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Player</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Rebounds</string>
          </property>
         </column>
         <column/>
         <column/>
         <column/>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_5">
        <attribute name="title">
         <string>Assists</string>
        </attribute>
        <widget class="QTableWidget" name="tableWidget_5">
         <property name="geometry">
          <rect>
           <x>105</x>
           <y>10</y>
           <width>321</width>
           <height>192</height>
          </rect>
         </property>
         <property name="rowCount">
          <number>20</number>
         </property>
         <property name="columnCount">
          <number>6</number>
         </property>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <column>
          <property name="text">
           <string>Rank</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Player</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Assists</string>
          </property>
         </column>
         <column/>
         <column/>
         <column/>
        </widget>
       </widget>
       <widget class="QWidget" name="rebounds">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>28</pointsize>
         </font>
        </property>
        <attribute name="title">
         <string>Players</string>
        </attribute>
        <widget class="QTableWidget" name="tableWidget">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>10</y>
           <width>211</width>
           <height>192</height>
          </rect>
         </property>
         <property name="rowCount">
          <number>5</number>
         </property>
         <property name="columnCount">
          <number>4</number>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <column>
          <property name="text">
           <string>Player</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Team</string>
          </property>
         </column>
         <column/>
         <column/>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Field Goal %</string>
        </attribute>
        <widget class="QTableWidget" name="tableWidget_6">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>10</y>
           <width>321</width>
           <height>192</height>
          </rect>
         </property>
         <property name="rowCount">
          <number>20</number>
         </property>
         <property name="columnCount">
          <number>6</number>
         </property>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <column>
          <property name="text">
           <string>Rank</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Player</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Field Goal %</string>
          </property>
         </column>
         <column/>
         <column/>
         <column/>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>                                        Games Board</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Home Team:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Visitor Team:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_2"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Year/ Season:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFontComboBox" name="fontComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget_2">
       <property name="rowCount">
        <number>6</number>
       </property>
       <property name="columnCount">
        <number>4</number>
       </property>
       <property name="Home Team" stdset="0">
        <string/>
       </property>
       <property name="Visitor Team" stdset="0">
        <string/>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column>
        <property name="text">
         <string>Home Team</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Visitor Team</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Season </string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Wins</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1459</width>
     <height>42</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
